getgather/mcp/americanairlines.py

```diff
@@ -9,8 +12,55 @@
 @americanairlines_mcp.tool
 async def get_upcoming_flights() -> dict[str, Any]:
     """Get upcoming flights of americanairlines."""
-    return await dpage_mcp_tool(
+
+    async def action(page: Page) -> dict[str, Any]:
+        try:
+            async with page.expect_response(
```

Comment:
I think we can simplify this by using `handle_network_extraction` from `actions.py`


```py
async def action(page: Page) -> dict[str, Any]:
  resp = handle_network_extraction(page, "/api/upcoming-trips")
  ....
```